rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -60,10 +59,8 @@ void opcode_0003(const ScriptArguments& args, const ScriptInt time) {
 	@arg arg2 
 */
 void opcode_0004(const ScriptArguments& args, ScriptIntGlobal arg1G, const ScriptInt arg2) {
-	RW_UNIMPLEMENTED_OPCODE(0x0004);
-	RW_UNUSED(arg1G);
-	RW_UNUSED(arg2);
 	RW_UNUSED(args);
+	arg1G = arg2;
```

Comment:
I'd have assumed that ScriptIntGlobals are like pointers? Are they a reference? If so, the name should suggest that.
It's hard to see that this changes something outside of this function.